http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeoAuthProvider.scala

```diff
@@ -24,6 +25,15 @@ object NotebookClusterActions {
   val allActions = Seq(GetClusterStatus, ConnectToCluster, SyncDataToCluster, DeleteCluster, StopStartCluster)
 }
 
+object PersistentDiskActions {
+  sealed trait PersistentDiskAction extends LeoAuthAction
```

Comment:
can we move `sealed trait PersistentDiskAction extends LeoAuthAction` outside the object itself.
For `allActions`, can you use `sealerate`
```
import ca.mrvisser.sealerate
sealerate.values[PersistentDiskActions]
```